src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/ClearAlternatesTag.cs

```diff
@@ -3,13 +3,12 @@
 using System.Threading.Tasks;
 using Fluid;
 using Fluid.Ast;
-using Fluid.Tags;
 
 namespace OrchardCore.DisplayManagement.Liquid.Tags
 {
-    public class ClearAlternatesTag : ExpressionTag
+    public class ClearAlternatesTag
     {
-        public override async ValueTask<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context, Expression expression)
+        public static async ValueTask<Completion> WriteToAsync(Expression expression, TextWriter writer, TextEncoder encoder, TemplateContext context)
```

Comment:
Some of these tags have really simple body and might benefit from the `return Awaited()` trick to remove async state machines, beneficial when tags are evaluated a lot and common case is that task is already resolved